presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeSimpleQueries.java

```diff
@@ -116,4 +119,17 @@ public void testJsonFileBasedFunction()
         assertQuery("SELECT json.test_schema.eq(1, linenumber) FROM lineitem", "SELECT 1 = linenumber FROM lineitem");
         assertQuery("SELECT json.test_schema.sum(linenumber) FROM lineitem", "SELECT sum(linenumber) FROM lineitem");
     }
+
+    @Test
+    public void testBroadcastJoin()
```

Comment:
Is this needed? Existing join tests should be sufficient, no?

Reply:
Existing should be sufficient. Removed.